cli/tools/coverage/mod.rs

```diff
@@ -654,6 +689,7 @@ pub async fn cover_files(
     None => None,
   };
 
+  let mut report_outputs: Vec<ReportOutput> = vec![];
   for script_coverage in script_coverages {
```

Comment:
Hmm. I probably messed up in the merge here and removed the borrow from `&script_coverages`.

Reply:
I think I have a spare borrow laying around